### ISSUE TITLE
Snap list icon

### ISSUE THREAD

**Issue by RubenVerg** [scratchblocks/scratchblocks#340]
Could you please add the list icon from Snap? It should render as a rectangle input with two orange bars in it.

**Comment by GrahamSH-LLK** [comment]
Yes, this would be great!

**Comment by GrahamSH-LLK** [comment]
@tjvr How could we implement this?

**Comment by tjvr** [comment]
I don't see an argument against this!

You'd need to add an icon in `style.js` for it, and hook it up elsewhere -- if you look for uses of `addInput` for example it should show how to do it.

**Comment by GrahamSH-LLK** [comment]
The problem is, I can’t actually find the icon itself, does anyone have it?

**Comment by CST1229** [comment]
> 
> 
> The problem is, I can’t actually find the icon itself, does anyone have it?

Looks like the icon is [actually drawn using code](https://github.com/jmoenig/Snap/blob/bf0680154f80c3c90f8fe2ee227ac1e2109ae05b/src/symbols.js#L2222-L2243).

**Comment by tjvr** [comment]
Should be easy enough for someone to translate those drawing commands into SVG, yeah!

**Comment by CST1229** [comment]
I have successfully traced both of the list icons.

![flat design (3.0)](https://assets.scratch.mit.edu/822050e2c028d8d255a5d4bcf31105ef.svg) ![normal design (2.0)](https://assets.scratch.mit.edu/afaaecedc41ddcc11966f2468f75cc58.svg)

Now all that's left is to add them... (and maybe fix some issues with them)

(also don't mind me using assets for github image hosting, githubusercontent doesn't support svg)

**Comment by tjvr** [comment]
What's the difference between them?

**Comment by CST1229** [comment]
> What's the difference between them?

The first one is used in flat design, second one is used in normal design in Snap_!_, I was thinking the flat one should be used for 3.0 style and the normal one for 2.0 style.

**Comment by tjvr** [comment]
@CST1229 We should be able to add a `@list` icon in just the same way you added `@stopSign`!

**Comment by CST1229** [comment]
> 
> 
> @CST1229 We should be able to add a `@list` icon in just the same way you added `@stopSign`!

Yes, only thing that's preventing me from doing that now is that I drew the icon in Scratch, which does weird things with positioning it when exported and messes it up a lot when directly used in scratchblocks.

**Comment by CST1229** [comment]
Now I have manually recreated the 2 list monitors in a text editor!

![flat](https://assets.scratch.mit.edu/492200b72feff3f4a9431bbe6ebbb213.svg) ![normal](https://assets.scratch.mit.edu/8aaa2a7ba8c94d150f9feb1c1509e028.svg)
